e2e/pages/administration/jobs/background/background.spec.js

```diff
@@ -0,0 +1,30 @@
+import LoginComponent from '../../../../components/login/login.po';
+import BackgroundPage from './background.po';
+import Hooks from '../../../../utilities/hooks';
+
+let background, hooks, loginComponent;
+
+beforeEach(async () => {
+    loginComponent = new LoginComponent();
+    background = new BackgroundPage();
+    hooks = new Hooks();
+    await hooks.open('#/search');
+});
+
+describe('ONC-Staff user can see', () => {
+    beforeEach(function () {
+        loginComponent.logInWithEmail('oncstaff');
+        hooks.open('#/administration/jobs/background');
+    });
+
+    it('correct background jobs', () => {
```

Comment:
`it` blocks be should in the form "should <fill in the blank>", so here I'd go with something like 
```suggestion
    it('should see the right set of background jobs', () => {
```